dcos/config.py

```diff
@@ -16,6 +17,7 @@ def load_from_path(path, mutable=False):
     :rtype: Toml | MutableToml
     """
 
+    util.ensure_dir_exists(os.path.dirname(path))
```

Comment:
This is fine if you want to get this out ASAP, but it'd be better to put the directory creation back in the env setup script.  We don't want to create directories if the user sets a custom DCOS_CONFIG to /non/existent/dir/path